modules/lib/lib-node/src/test/resources/test/FindNodesByQueryHandlerTest.js

```diff
@@ -0,0 +1,66 @@
+var nodeLib = require('/lib/xp/node');
+var assert = require('/lib/xp/testing');
+
+var repo = nodeLib.connect({
+    repoId: 'com.enonic.cms.default',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692790)
```suggestion
    repoId: "com.enonic.cms.default",
```